lib/src/app/feature/authentication/presentation/login_screen.dart

```diff
@@ -156,116 +160,184 @@ class _LoginScreenState extends State<LoginScreen> {
-                                ),
-                                const Icon(Icons.arrow_drop_down),
-                              ],
-                            ),
-                          ),
-                        ),
+                    Padding(
+                      padding: EdgeInsets.only(
+                        top: MediaQuery.of(context).padding.top,
+                        left: width / 10,
+                        right: width / 10,
                       ),
-                    ),
-                    Flexible(
-                      flex: 7,
-                      child: SizedBox(
-                        height: height / 13,
-                        child: TextFormField(
-                          decoration: const InputDecoration(
-                            hintText: 'Phone number',
-                          ),
-                          controller: _controller,
-                          cursorWidth: 3,
-                          style: Theme.of(context)
-                              .textTheme
-                              .headline3
-                              ?.copyWith(color: Styleguide.main),
-                          keyboardType: TextInputType.phone,
+                      child: SingleChildScrollView(
+                            child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.center,
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            SizedBox(
+                              height: height / 7,
+                            ),
+                            Padding(
+                              padding: EdgeInsets.symmetric(
+                                  horizontal: width * 0.05),
+                              child: const Logo(),
+                            ),
+                            SizedBox(
+                              height: height * 0.1,
+                            ),
+                            Container(
+                              decoration: BoxDecoration(
+                                borderRadius: BorderRadius.circular(100),
+                              ),
+                              height: height / 13,
+                              width: 8 * (width / 10),
+                              padding: const EdgeInsets.all(5),
+                              child: Row(
+                                mainAxisAlignment: MainAxisAlignment.center,
+                                crossAxisAlignment: CrossAxisAlignment.center,
+                                children: [
+                                  Flexible(
+                                    flex: 2,
+                                    child: FittedBox(
+                                      child: Material(
+                                        child: InkWell(
+                                          onTap: () async {
```

Comment:
I see `async` is not added in this PR and is not related to the task of this PR, but can we remove it if not required? Or should we do such unrelated "clean-up" changes (this one and others if any) in separate PR ideally? Let me know your thoughts.